src/internal/components/series-marker/styles.scss

```diff
@@ -18,7 +18,7 @@ $marker-margin-right: cs.$space-static-xxs;
   border-start-end-radius: 2px;
   border-end-start-radius: 2px;
   border-end-end-radius: 2px;
-  inline-size: $marker-size;
+  display: flex;
```

Comment:
Note: there is a test failing because apparently the file `src/internal/components/series-marker/index.tsx` cannot load the stylesheet file generated by this file anymore (but not sure why TBH): https://github.com/cloudscape-design/chart-components/actions/runs/19270938423/job/55185308913?pr=124